### PR TITLE
Feat book audit

### PR DESCRIPTION
**Pull request by nathanss**
This PR creates both the mock service to represent "Secretaria do meio-ambiente" in route "/secretariat" and consume this service in order to create the screen to allow the user to book an audit.

To test, access "Agendar auditoria" from the Menu.

User Stories: https://trello.com/c/J1K2naX3
https://trello.com/c/ke29sviX

### CODE REVIEW

**Comment by isaqueha** [comment]
When trying to access the http://localhost:3000/audits/new route, the system is throwing an error.
Probably a missing parameter.

image

File: views/bookAudit.ejs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```
Comment (isaqueha) [inline views/bookAudit.ejs:1]: Don't we need to show/use Status for the Audit?
This is mentioned in the US: 
image

File: views/newaudit.ejs [hunk 2]
```diff
@@ -49,6 +49,19 @@
           <% }) %>
         </select>
       <% } %>
+
+      <p>Datas</p>
+      <% if (availableHoursList && availableHoursList.length) { %>
```
Comment (isaqueha) [inline views/newaudit.ejs:54]: I think that this is what breaks when entering the newAudit route.
Please populate `availableHoursList` in the `audits.js` when navigating there.

File: routes/audits.js [hunk 3]
```diff
@@ -23,14 +23,54 @@ router.get('/new', async function(req, res) {
       title: 'Add New Audit', 
       action: "/audits/add" ,
       projectlist: docs,
-      requirementlist: requirementlist,
```
Comment (isaqueha) [inline routes/audits.js:26]: Just a question:
Should we adopt a pattern here and everywhere when passing parameters to a route?

1. `requirementlist: requirementlist,`
or just
2. `requirementlist,`

File: routes/audits.js [hunk 4]
```diff
@@ -23,14 +23,54 @@ router.get('/new', async function(req, res) {
...
 });
 
-/* GET Edit Audit page. */
+router.get('/book', async function(req, res) {
+  const db = req.db;
+  const requirementcollection = db.get('requirementcollection');
+  const projectCollection = db.get('projectcollection');
+  const requirementlist = await requirementcollection.find({});
+  const availableHoursList = await getAvailableHours(req);
+  const docs = await projectCollection.find({});
```
Comment (isaqueha) [inline routes/audits.js]: Maybe `docs` could be renamed to just `projectlist`.

File: routes/audits.js [hunk 5]
```diff
@@ -78,7 +119,7 @@ router.post('/add', function(req, res) {
   });
 });
 
-/* POST to Update Audit Service */
+
```
Comment (isaqueha) [inline routes/audits.js]: Extra empty space here

File: views/bookAudit.ejs (hunk 1 above)
Comment (isaqueha) [reply views/bookAudit.ejs:1]: Maybe we can set the status for the Audit in the `audits.js` when booking it.

**Review by isaqueha — CHANGES_REQUESTED** [review]
Some suggestions

File: views/bookAudit.ejs (hunk 1 above)
Comment (nathanss) [reply views/bookAudit.ejs:1]: yes!

File: routes/audits.js (hunk 3 above)
Comment (nathanss) [reply routes/audits.js:26]: Yes, I am adopting the second one.

Comment (nathanss) [reply routes/audits.js:26]: Please tell me if you see more of these.

File: routes/audits.js (hunk 5 above)
Comment (nathanss) [reply routes/audits.js]: okk

File: routes/audits.js (hunk 4 above)
Comment (nathanss) [reply routes/audits.js]: yes

**Review by isaqueha — APPROVED** [review]
NOooice code

**Pull request merged**